library/modules/Textures.cpp

```diff
@@ -292,13 +368,28 @@ static void uninstall_reset_point() {
     INTERPOSE_HOOK(tracking_stage_new_arena, logic).remove();
 }
 
+static void reserve_static_range() {
+    reserved_range.init(enabler->textures.init_texture_size);
```

Comment:
we should check to make sure that the number of current textures == enabler->textures.init_texture_size. if not, we're in an invalid state and the reserved range cannot be initialized. in this case, we should print a warning and continue, and all reserved textures should fall back to becoming dynamic